examples/prom/nonlinear_elasticity_global_rom.cpp

```diff
@@ -764,23 +826,30 @@ int main(int argc, char* argv[])
                 basisFileName + "_H");
     }
 
-    RomOperator* romop = 0;
+    RomOperator *romop = 0;
 
-    const CAROM::Matrix* BV_librom = 0;
-    const CAROM::Matrix* BX_librom = 0;
-    const CAROM::Matrix* H_librom = 0;
-    const CAROM::Matrix* Hsinv = 0;
+    const CAROM::Matrix *BV_librom = 0;
+    const CAROM::Matrix *BX_librom = 0;
+    const CAROM::Matrix *H_librom = 0;
+    const CAROM::Matrix *Hsinv = 0;
 
     int nsamp_H = -1;
 
-    CAROM::SampleMeshManager* smm = nullptr;
+    CAROM::SampleMeshManager *smm = nullptr;
```

Comment:
There are several pointers here that need to be deleted, alternatively they can be `std::unique_ptr`.